addon/run-task.js

```diff
@@ -4,7 +4,7 @@ import { assert } from '@ember/debug';
 import getTask from './utils/get-task';
 import { registerDisposable } from './utils/disposable';
 
-const { WeakMap } = Ember;
+const { WeakMap, deprecate } = Ember;
```

Comment:
Can we import `deprecate` from `@ember/debug`? We're already importing `assert`.